### PR TITLE
Fix building crc32_arm64 on NetBSD/aarch64

### PR DESCRIPTION
**Pull request by AlarixBSD**
pmull_supported is not necessarily defined before crc32c_aarch64

All of my contributions are freely available under the new BSD license

### CODE REVIEW

**Review by dr-m — COMMENTED** [review]
I think that this needs to be submitted against the earliest applicable branch (10.5 instead of 10.6).

**Comment by alarixbsd** [comment]
Ooops - also added a note, this change is under the new BSD license.

File: mysys/crc32/crc32_arm64.c [hunk 1]
```diff
@@ -2,13 +2,13 @@
 #include <string.h>
 #include <stdint.h>
 
+static int pmull_supported;
+
 #if defined(HAVE_ARMV8_CRC)
```
Comment (dr-m) [inline mysys/crc32/crc32_arm64.c:7]: Could the definition be moved after the `#ifdef HAVE_ARMV8_CRC`? As far as I remember, we would only use the `pmull` instruction in combination with CRC instructions, not by itself.

**Review by dr-m — COMMENTED** [review]
@grooverdan Please complete the review, check our ARMv8 builders, and merge this. I will be on vacation for the next couple of weeks.

File: mysys/crc32/crc32_arm64.c (hunk 1 above)
Comment (alarixbsd) [reply mysys/crc32/crc32_arm64.c:7]: pmull_supported is referenced in crc32c_aarch64 outside of the HAVE_ARMV8_CRC block, which causes the build error in the first place.

**Comment by AlarixBSD** [comment]
ping?

**Comment by grooverdan** [comment]
ack. I haven't forgotten. I just have a couple of fixed deadline tasks to resolve first. Will probably be early next week.

**Comment by grooverdan** [comment]
So this fixes the compile error, but on NetBSD the sysctl hw tree doesn't support enough detailed information to provide an alternate  `crc32c_aarch64_available` implementation.

**Comment by grooverdan** [comment]
Thanks @AlarixBSD

**Pull request merged**